README.md

```diff
@@ -66,6 +68,7 @@ deploy:
     - Defaults to `coding-pages` on Coding.net.
     - Otherwise defaults to `master`.
   - **token**: Optional token value to authenticate with the repo. Prefix with `$` to read token from environment variable (recommended). Repo must be a http(s) url. [More details](#deploy-with-token).
+  - **token_name** coding.net needs token name and token to authenticate. [More details](#deploy-with-token)
```

Comment:
Should mentioned that the configuration only valid for `coding.net`.

Reply:
fixed